src/sharers/whatsapp.js

```diff
@@ -7,5 +8,5 @@ export default function whatsapp(options = {}) {
     text: [title, url].filter(item => item).join(' '),
   });
 
-  return window.location.assign(`whatsapp://send?${params}`);
+  return mobileShare(`whatsapp://send?${params}`);
```

Comment:
You need to check mobileShare call instead of location.assign in the sharers/whatsapp test